src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -194,11 +200,16 @@ def _snap_to_rail(string: str, rails: Set[str]) -> str:
     rails_list = list(rails)
     rail = _extract_rail(processed_string, rails_list[0], rails_list[1])
     if not rail:
+        if verbose:
+            print(f"- Cannot snap {repr(string)} to rails: {rails}")
```

Comment:
Might be cleaner if we just added a utility like this:

https://github.com/run-llama/llama_index/blob/480ef30d23fe0395bff93c2d383687a73cea702c/llama_index/readers/github_readers/utils.py#L19

Reply:
great yeah, I think I've already made that it's just not pushed up at the moment